src/components/ebay-snackbar-dialog/examples/03-stacked/template.marko

```diff
@@ -0,0 +1,20 @@
+<ebay-snackbar-dialog open=state.open windowClass="snackbar-dialog__window--column" type="action" on-close("closeSnackbar")>
```

Comment:
Rather than `windowClass`, can we add a `layout` attribute? Values should be `row` (default) or `column`.

Reply:
Window class should be a private method. This should be in the snackbar template.